scripts/test_updates_pg14.sh

```diff
@@ -3,6 +3,7 @@
 set -e
 
 SCRIPT_DIR=$(dirname $0)
+export PG_VERSION=14
```

Comment:
If you set this unconditionally here you should remove PG_VERSION env from caller in .github/workflows/update-test.yaml. Also this is unrelated to your change so i'd prefer to see this in separate commit.